jormungandr/src/settings/start/mod.rs

```diff
@@ -153,13 +154,22 @@ impl RawSettings {
         }
     }
 
+    fn rpc_config(&self) -> Option<Rpc> {
+        let cmd_listen_opt = self.command_line.rpc_arguments.listen;
+        match cmd_listen_opt {
+            Some(cmd_listen) => Some(Rpc { listen: cmd_listen }),
+            None => None,
+        }
```

Comment:
Small style thing: could this become: `cmd_listen_output.map(|listen| Rpc { listen })`?